config/connect-mirror-maker.properties

```diff
@@ -57,3 +57,6 @@ config.storage.replication.factor=1
 # replication.policy.separator = _
 # sync.topic.acls.enabled = false
 # emit.heartbeats.interval.seconds = 5
+
+# enable flow in the logs for improved diagnostics
+add.flow.context = true
```

Comment:
Nit: this should be false according to your KIP but I guess you haven't updated this PR since, so just making a note.

Reply:
The default of the config is false, but I think that is only necessary for being backward compatible. I honestly think that the recommendation (the sample config) should be to enable the extra log information by default, as many use-cases will benefit from it.

Reply:
I left my thoughts on the discussion thread, but TL;DR: I think disabling by default both programmatically and in relevant config files is the safest (and follows precedent set by prior KIPs to added connector contexts to log files), and we can add a commented-out line to relevant config files that would enable the new log context behavior and add an explanation above it with our recommendations to users to un-comment it if they're starting new clusters or if their logging infrastructure is prepared to handle the changes it would cause.